### PR TITLE
add note to User-Auth with LDAP doc

### PR DESCRIPTION
**Pull request by mschreml**
Issue: #2774

### CODE REVIEW

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 1]
```diff
@@ -316,6 +316,8 @@ y groups found::
   This tells you approximately how many groups will be available in ownCloud. 
   The number updates automatically after any change.
 
+NOTE: Renaming of LDAP-Groups is not supported. Changes in LDAP are not propagated to ownCloud.
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:319]:
```suggestion
NOTE: Renaming of LDAP-Groups on the LDAP Server is not supported. Changes like renaming in LDAP will not be propagated to ownCloud.
```

**Comment by mmattel** [comment]
Has this issue been reported to core or ldap?
@jvillafanez or @butonic, can you take a look on the source side?

**Review by phil-davis — COMMENTED** [review]
The words are fine, and I believe that this is true! (And I think it has always been true)
Can "someone" confirm that there has always been this restriction?
And what does happen if the LDAP group is renamed? Does the "old" group name in ownCloud keep working but never get any more member updates? Or?

**Comment by mmattel** [comment]
Usually an LDAP Group Name has a GID in the backend. The question is, which one was used and how...

**Comment by voroyam** [comment]
> And what does happen if the LDAP group is renamed? Does the "old" group name in ownCloud keep working but never get any more member updates? Or?

Old group name keeps working.

**Review by voroyam — APPROVED** [review]

**Pull request merged**